src/memcached.c

```diff
@@ -285,6 +297,47 @@ static void submit_gauge2(const char *type, const char *type_inst,
   plugin_dispatch_values(&vl);
 }
 
+static gauge_t calculate_rate(gauge_t part, gauge_t total, gauge_t *prev_part,
+                              gauge_t *prev_total) {
+  if (isnan(*prev_part) || isnan(*prev_total) || (part < *prev_part) ||
+      (total < *prev_total)) {
+    *prev_part = part;
+    *prev_total = total;
+    return NAN;
+  }
+
+  gauge_t num = part - *prev_part;
+  gauge_t denom = total - *prev_total;
+
+  *prev_part = part;
+  *prev_total = total;
+
+  if (num == 0 || denom == 0)
+    return 0;
+
+  return 100.0 * num / denom;
```

Comment:
Either rename this function to `…_ratio` and return `num / denom` here, or use `…_ratio_percent` and return `100.0 * num / denom`.

Reply:
Renamed to `..._ratio_percent`.

Reply:
Renamed to `.._ratio_percent`.